scripts/ldev2pcs

```diff
@@ -102,7 +102,7 @@ def configure_location(resource, nodes, fixed_score_string=None):
             score -= 10
 
 def configure():
-    cmd = 'pcs cluster setup --force --local'
+    cmd = 'pcs cluster setup --force'
```

Comment:
Removing `--local` makes sense to me. I was reading the man page for `pcs cluster setup` and it appears that `--force` is not a supported flag for that command. If `--force` is not necessary, then perhaps we should remove it as well?